src/data/_contributors.yaml

```diff
@@ -1605,6 +1605,21 @@ rupl:
   description:
     en: Chris is a frontend developer and web performance advocate.
 
+nsatragno:
+  name:
+    given: Nina
+    family: Satragno
+  org:
+    name: Google
+    unit: Chrome
+  country: CA
+  role:
+    - software engineer
```

Comment:
I think `role` is supposed to be something like `author` (i.e. the role in the context of WebFu). The `description` is where the job title (or any preferred title) can go